### PR TITLE
Document the target phase in “Introduction to events”

### PR DESCRIPTION
**Pull request by sideshowbarker**
Fixes https://github.com/mdn/content/issues/6657

### CODE REVIEW

File: files/en-us/learn/javascript/building_blocks/events/index.html [hunk 1]
```diff
@@ -518,7 +518,7 @@ <h6 id="Hidden_video_example">Hidden video example</h6>
 
 <h4 id="Bubbling_and_capturing_explained">Bubbling and capturing explained</h4>
 
-<p>When an event is fired on an element that has parent elements (in this case, the {{htmlelement("video")}} has the {{htmlelement("div")}} as a parent), modern browsers run two different phases — the <strong>capturing</strong> phase and the <strong>bubbling</strong> phase.</p>
+<p>When an event is fired on an element that has parent elements (in this case, the {{htmlelement("video")}} has the {{htmlelement("div")}} as a parent), modern browsers run three different phases — the <strong>capturing</strong> phase, the <strong>target phase</strong>, and the <strong>bubbling</strong> phase.</p>
```
Comment (teoli2003) [inline files/en-us/learn/javascript/building_blocks/events/index.html:522]: Nit: take the second _phase_ out of the `<strong>` element

File: files/en-us/learn/javascript/building_blocks/events/index.html [hunk 2]
```diff
@@ -527,7 +527,13 @@ <h4 id="Bubbling_and_capturing_explained">Bubbling and capturing explained</h4>
  <li>Then it moves on to the next element inside <code>&lt;html&gt;</code> and does the same thing, then the next one, and so on until it reaches the element that was actually selected.</li>
 </ul>
 
-<p>In the <strong>bubbling</strong> phase, the exact opposite occurs:</p>
+<p>In the <strong>target</strong> phase:</p>
+<ul>
+ <li>The browser calls any event listeners registered for this phase (which is the point at which the event has arrived at {{domxref("EventTarget", "the event's target", "", 1)}}).</li>
+ <li>Then, if {{domxref("Event.bubbles")}} is <code>true</code>, it propagates the event to the next immediate ancestor element, then the next one, and so on until it reaches the <code>&lt;html&gt;</code> element. Otherwise, if {{domxref("Event.bubbles")}} is <code>false</code>, it stops and doesn’t propagate the event to any ancestors of the target.</li>
```
Comment (teoli2003) [inline files/en-us/learn/javascript/building_blocks/events/index.html:533]: I would write something like "it doesn't propagate. (...) the event and stops." as stopping is the consequence of not bubbling

**Review by teoli2003 — COMMENTED** [review]
2 nits, and it is great to have this fixed.

File: files/en-us/learn/javascript/building_blocks/events/index.html (hunk 1 above)
Comment (sideshowbarker) [reply files/en-us/learn/javascript/building_blocks/events/index.html:522]: oops — thanks for catching that, fixed now

File: files/en-us/learn/javascript/building_blocks/events/index.html (hunk 2 above)
Comment (sideshowbarker) [reply files/en-us/learn/javascript/building_blocks/events/index.html]: OK, updated

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**